nssdb/certificate_test.go

```diff
@@ -167,12 +181,16 @@ func TestNSSDB_Import(t *testing.T) {
 			if err != nil {
 				t.Skip("pk1sign not installed")
 			}
+
 			db, dir := v.connectDir(t)
+			require.NoError(t, db.Reset(ctx))
```

Comment:
This is why the `pk1sign` test was passing for the IV without the 040e prefix - because it was testing with a key imported with `pk12util` not the one imported by this package.